pkg/platform/platform.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package platform
+
+import "istio.io/istio/pkg/env"
+
+type PlatformType string
+
+const (
+	Empty     PlatformType = ""
```

Comment:
nit: Should we remove this `Empty` enum? Since I think unset variable maybe enough to represent a "nothing specical" state, and it is documented.

Reply:
Or name it `Default` or `Vanilla`?

Reply:
I liked `Default`.